backend/src/services/integration-auth/integration-sync-secret.ts

```diff
@@ -557,6 +557,8 @@ const syncSecretsAWSParameterStore = async ({
                 `AWS Parameter Store Error [integration=${integration.id}]: double check AWS account permissions (refer to the Infisical docs)`
               );
             }
+
+            throw err;
```

Comment:
context: I previously introduced a new set of policies that have to be added for the AWS integration to work as intended. I explicitly did not throw an error here so that the flow does not break for users that have not yet adjusted their policies to include the new set of permissions. Maybe we could adjust this so that it only throws an error if its not an "AccessDeniedException"?

i should have added a comment

Reply:
Oh gotcha, great catch in that case. I've updated the code to only throw if it's an access denied error!

Reply:
oh wait, we should only be throwing if it's not an access denied error